File: 240308/개수 세기/count-numbers.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {

    int n,m;

    cin>>n>>m;

    int arr[100]={};
    int cnt=0;

    for(int i=0;i<n;i++){
        cin>>arr[i];
        if(arr[i]==m) cnt++;
    }
    cout<<cnt;
    return 0;
}

File: 240605/개수 세기/count-numbers.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
    
    Scanner sc=new Scanner(System.in);
    int n=sc.nextInt();
    int m=sc.nextInt();

    int[] arr=new int [100];
    int cnt=0;

    for(int i=0;i<100;i++){
        arr[i]=sc.nextInt();
        if(arr[i]==m)
        cnt++;
    }
    System.out.print(cnt);
    }
}